eng/packages/http-client-csharp-mgmt/emitter/src/index.ts

```diff
@@ -1,4 +1,4 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-export * from "./emitter.js";
+export { $lib } from "./lib/lib.js";
```

Comment:
we are planning to have another CDK emitter based on MPG emitter, shall we export `$onEmit` as well?

Reply:
Sure, but we can just do that as it is needed.